zenoh/src/net/routing/interceptor/access_control.rs

```diff
@@ -278,7 +278,49 @@ impl InterceptorTrait for IngressAclEnforcer {
                     return None;
                 }
             }
-            _ => {}
+            NetworkBody::Response(Response { .. }) => {
+                if self.action(AclMessage::Reply, "Reply (ingress)", key_expr?) == Permission::Deny
+                {
+                    return None;
+                }
+            }
+            // Unfiltered Delete messages
+            NetworkBody::Push(Push {
+                payload: PushBody::Del(_),
```

Comment:
`Delete` should be handled in the same way a `Put` is handled. We should also add it as part of the config.

Reply:
Yes, it does 👍